### ISSUE TITLE
Specify build dependencies

### ISSUE THREAD

**Issue by totchi-lagawi** [Eliyaan/U#1]
Sokol depends on local libraries, which are listed below :
| System              | Installation method                                                                                |
|---------------------|----------------------------------------------------------------------------------------------------|
| Debian/Ubuntu based | `sudo apt install libxi-dev libxcursor-dev libgl-dev libasound2-dev`                               |
| Fedora/RH/CentOS    | `sudo dnf install libXcursor-devel libXi-devel libX11-devel libglvnd-devel`                        |
| NixOS               | add `xorg.libX11.dev xorg.libXcursor.dev xorg.libXi.dev libGL.dev` to `environment.systemPackages` |

Not having them cause C compilation errors, and V doesn't say that Sokol depends on those...

**Comment by Eliyaan** [comment]
They do say it on the V readme : https://github.com/vlang/v?tab=readme-ov-file#testing-and-running-the-examples but I guess that's where the list comes from.
When you compile doesn't it say something like: `libxcursor... missing`  ?

**Comment by totchi-lagawi** [comment]
No, there's only a message saying that it's a V compiler bug that should be reported, while it's not. It would be great to add it to the `README.md`, to not confuse beginners trying your project.

**Comment by Eliyaan** [comment]
I'll ask in the V server if they can do something about it. Because it is a problem for all gg projects and it is the main graphics module.

**Comment by totchi-lagawi** [comment]
Yeah, if they want to create a beginner-friendly language it's better to have beginner-friendly error messages...
(I'm not criticising V, don't punch me lmao)

**Comment by Eliyaan** [comment]
Yes I agree, I asked. They will see if it is possible
(dont worry, it is nice to find new ways to help begginers and I dont punch anyone)

**Comment by totchi-lagawi** [comment]
It was just a joke... Whatever, I hope they'll add the message.
